R/importWQP.R

```diff
@@ -160,12 +109,26 @@ importWQP <- function(obs_url, zip = TRUE, tz = "UTC",
 #' 
 parse_WQP <- function(retval, tz = "UTC"){
```

Comment:
We have always advertised that when we convert to dateTime objects, we're doing it by default to UTC. If a user wants a different time zone, they would put that timezone in this "tz" argument here. We can't pick something better because users can (and do) request data that span time zones (and a vector of POSIXct in R can only have 1 timezone attribute, it's on the vector not the individual dateTime).